README.md

```diff
@@ -4,28 +4,33 @@
 [![codecov](https://codecov.io/gh/openshift/backplane-cli/branch/main/graph/badge.svg)](https://codecov.io/gh/openshift/backplane-cli)
 [![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
-backplane-cli is a CLI tool to interact with backplane api.
+backplane-cli is a CLI tool to interact with [backplane api](https://github.com/openshift/backplane-api).
+
+The onboarding documentation can be found on [The Source Wiki](https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_user_documentation).
 
 ## Install
 
 ### Option 1: Use `go install`
 
-You need to have `go` installed, if you want to use this method,
-the minimal version required is `go 1.18`. You can check this with `go version`.
+For using this method, you need to have `go` installed.
+
+The minimal version required is `go 1.18`. You can check this with `go version`.
 
 Then run `go install github.com/openshift/backplane-cli/cmd/ocm-backplane@{TAG_VERSION}`
 where `TAG_VERSION` is the version you wish to install.
 
 Example: `go install github.com/openshift/backplane-cli@0.0.8`
 
-`go install` will fetch, build binary and install them to your $GOBIN if set, or $GOPATH/bin,
-you should and move this binary onto your $PATH if desired.
+`go install` will fetch, build binary and install that to your $GOBIN (if set) or $GOPATH/bin,
+you should move this binary onto your $PATH if desired.
```

Comment:
This won't be necessary, usually, if go environment structure is configured correctly, `GOBIN` will add the binaries to `PATH`
```suggestion
```